### ISSUE TITLE
- Dropdown - Accessibility

### ISSUE THREAD

**Issue by awusuwah** [awusuwah/marko-boilerplate#2]
To make this component as accessible as possible, it needs some general a11y improvements. Since I don't really have a clue what attributes are missing for an improved a11y experience, I will need to do some research and run a Lighthouse test or something.

**Comment by awusuwah** [comment]
I have added roles, aria-expanded, aria-controls to various elements in the dropdown.

I have also figured out that I can enable Accessibility Checking in the "Elements" tab of the chrome devtools. I don't fully understand the data in that view, but it allows me to see how far I am in terms of accessibility. I have also added some a11y features to the **Number** component.
